lms/djangoapps/teams/static/teams/js/views/my_teams.js

```diff
@@ -0,0 +1,32 @@
+;(function (define) {
+    'use strict';
+
+    define(['backbone', 'gettext', 'teams/js/views/teams'],
+        function (Backbone, gettext, TeamsView) {
+            var MyTeamsView = TeamsView.extend({
+                className: 'my-teams',
+
+                render: function() {
+                    TeamsView.prototype.render.call(this);
+                    if (this.collection.length === 0) {
+                        this.$el.append('<p>' + gettext('You are not currently a member of any teams.') + '</p>');
```

Comment:
@catong Can you give me a quick doc review on this string. You can see it when accessing the "My Teams" tab as a user who does not belong to any teams. Try it on my server:

http://andy-armstrong.sandbox.edx.org/courses/course-v1:edX+DemoX+Demo_Course/teams/#my-teams